lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -1,130 +1,174 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
+
 # Copyright: Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: postgresql_ext
-short_description: Add or remove PostgreSQL extensions from a database.
+short_description: Add or remove PostgreSQL extensions from a database
 description:
-   - Add or remove PostgreSQL extensions from a database.
+- Add or remove PostgreSQL extensions from a database.
 version_added: "1.9"
 options:
```

Comment:
@gundalow , yes, I see. However the common documentation is not modern. It should be updated before.
To avoid mixing all in this PR, I suggest that I'll note for me (and make notice in the working group in Ideas section) that this documentation needs to be updated by particular PR.
After that we might fix all pg modules by a small simple PR for each.
What do you think?

Reply:
That sounds totally fine. A single PR which updates all PostgreSQL modules to fix this would be good.